invocations/hcppipelines-4.0.0-1_invocation.json

```diff
@@ -0,0 +1,7 @@
+{
+	"bids_dir": "/bids",
+	"output_dir": "/output",
+	"analysis_level": "participant",
+	"license_key": "/license/license.txt",
+	"n_cpus":4
```

Comment:
```suggestion
	"n_cpus":4,
  	"stages":["PreFreeSurfer","FreeSurfer","PostFreeSurfer"]
```

Reply:
not this one, there's already an anatomical hcp pipelines in the repo. This is meant to run everything.

Reply:
ohhh k